src/DeploymentProvider/Providers/WebAppDeploymentProvider.ts

```diff
@@ -67,4 +68,13 @@ export class WebAppDeploymentProvider extends BaseWebAppDeploymentProvider {
             core.debug(`Skipped updating appCommandLine. Current value is: ${currentStartupCommand}`);
         }
     }
+
+    public async UpdateDeploymentStatus(isDeploymentSuccess: boolean) {
+        if(!!this.appService) {
+            await addAnnotation(this.actionParams.endpoint, this.appService, isDeploymentSuccess);

```

Comment:
What does this method do? Do we add annotation this currently also (without this fix)?

Reply:
Yes without this fix base class's UpdateDeploymnetStatus method was getting called 

https://github.com/Azure/webapps-deploy/blob/4fa2c3603a391d8e5417cdc43b70626bada9bc37/src/DeploymentProvider/Providers/BaseWebAppDeploymentProvider.ts#L44

I have overridden that method in this class by removing updateDeploymentStatus part

Reply:
When you create WebApp on azure portal you can enable [application insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview) for your app. Application insights monitors your app it has all logs of health of your app. this addAnnotation method just adding deployment success log to application insights if its enabled for your web app